### ISSUE TITLE
Adding images to docs site

### ISSUE THREAD

**Issue by anchit-chandran** [rcpch/epilepsy12-documentation#20]
TL/DR:
1. Upload all the images somewhere, I'll put them into repo
2. To embed them in the text, if you have an image called `cat.jpg`, you type:
3. `![Cat pic](../_assets/_images/how_to_edit_imgs/cat.jpg)`
4. [Example on docs site](https://github.com/rcpch/epilepsy12-documentation/blob/0a9d00b559ad174707b2b7469cb35771a57fae17/docs/admin-users/how-to-edit-docs.md?plain=1#L62C15-L62C68)

---

I'll have to upload the images to the repository to be shown on the website. If you send me a link to all of them hosted on something like dropbox/GDrive, I can upload all in one.

What would be very helpful is if you still link(*embed* in markdown) the images in your writing.

Say you have an image called `cat.jpg`. To embed it in markdown, you would type:

`![picture of a cat](cat.jpg)`

Which is the following structure:

`![IMAGE CAPTION](LINK TO IMAGE)`.

Specifically for our docs site, I will upload the images to an `_assets/images/` folder, so for your `cat.jpg` embed, you should write:

`![Cat pic](../_assets/_images/how_to_edit_imgs/cat.jpg)`.

You can see an example from the docs site of a screenshot I've uploaded in the 'how to edit docs' section: https://github.com/rcpch/epilepsy12-documentation/blob/0a9d00b559ad174707b2b7469cb35771a57fae17/docs/admin-users/how-to-edit-docs.md?plain=1#L62C15-L62C68

These GH Issues messages support markdown so this should load too (embedded by typing `![picture of a cat](https://i.natgeofe.com/n/548467d8-c5f1-4551-9f58-6817a8d2c45e/NationalGeographic_2572187_3x4.jpg)`:

![picture of a cat](https://i.natgeofe.com/n/548467d8-c5f1-4551-9f58-6817a8d2c45e/NationalGeographic_2572187_3x4.jpg)

**Comment by anchit-chandran** [comment]
Thanks those have been added so once you make the PR, should hopefully be able to see them
